src/main/java/com/wynntils/modules/utilities/configs/OverlayConfig.java

```
@@ -139,6 +139,9 @@ public enum ManaTextures {
     public static class ManaTimer extends SettingsClass {
         public static ManaTimer INSTANCE;
 
+        @Setting(displayName = "Force Hide Mana Timer", description = "Enabling this will hide the mana timer regardless of overlay config status.")
```

why isn't this just a normal toggle instead of being inverted (enabled = no timer)? and it wouldn't be disabled by default now, which was important

---

I assumed the default hide isn't important anymore now that there are multiple ways to disable it

---

i'd expect the majority of users to not really benefit from this feature, so it's best to have it disabled by default regardless